lib/scsi_defs.h

```diff
@@ -80,5 +83,37 @@ namespace scsi_defs {
     uint8_t vendorSpecific : 2;
   } ControlByte;
 
+  // SCSI Reference Manual Table 73 https://www.seagate.com/files/staticfiles/support/docs/manual/Interface%20manuals/100293068j.pdf
+  // All fields big-endian
+  struct ModeSense6Command {
+    OpCode opCode = OpCode::kModeSense6;
+    uint8_t reserved_1 : 4;
+    bool dbd : 1;
+    uint8_t reserved_2 : 3;
+    uint8_t pc : 2;
+    uint8_t page_code : 6;
+    uint8_t sub_page_code : 8;
+    uint8_t alloc_length : 8;
+    ControlByte control_byte;
+  } ABSL_ATTRIBUTE_PACKED;
+  static_assert(sizeof(ModeSense6Command) == 6);
+
+  // SCSI Reference Manual Table 75 https://www.seagate.com/files/staticfiles/support/docs/manual/Interface%20manuals/100293068j.pdf
+  // All fields big-endian
```

Comment:
I added a comment for this on the top of the namespace in #20. So we can remove this particular comment from here as #20 will take care of it

Reply:
Good catch, I forgot to remove this